### PR TITLE
[KAT-938] Rename python classes

### PR DESCRIPTION
**Pull request by arthurp**
With: https://github.com/KatanaGraph/katana-enterprise/pull/1682

### CODE REVIEW

File: python/katana/__init__.py [hunk 1]
```diff
@@ -27,14 +27,15 @@
...
+    from katana.katana_static_state import get_active_threads, set_active_threads, set_busy_wait
 
-    __version__ = katana.galois.get_version()
+    __version__ = katana.katana_static_state.get_version()
 
     from katana._loops import OrderedByIntegerMetric, PerSocketChunkFIFO, UserContext, do_all, for_each
     from katana.loop_operators import do_all_operator, for_each_operator, obim_metric
 except ModuleNotFoundError as e:
-    if "katana.galois" in str(e):
+    if "katana.katana_static_state" in str(e):
```
Comment (gurbinder533) [inline python/katana/__init__.py:38]: Any particular reason for calling this "katana_static_state"?

File: python/katana/local/_graph.pyx [hunk 2]
```diff
@@ -300,8 +300,8 @@ cdef class PropertyGraph(PropertyGraphBase):
             self._underlying_property_graph = handle_result_PropertyGraph(_PropertyGraph.Make(path_str, opts))
 
     @staticmethod
-    cdef PropertyGraph make(shared_ptr[_PropertyGraph] u):
-        f = <PropertyGraph>PropertyGraph.__new__(PropertyGraph)
+    cdef Graph make(shared_ptr[_PropertyGraph] u):
```
Comment (gurbinder533) [inline python/katana/local/_graph.pyx:303]: _Graph??

File: python/katana/local/_graph.pyx [hunk 3]
```diff
@@ -315,29 +315,29 @@ cdef class PropertyGraph(PropertyGraphBase):
         :param chunk_size: Chunk size for in memory representations during conversion. Generally this term can be
             ignored, but it can be decreased to reduce memory usage when converting large inputs.
         :type chunk_size: int
-        :returns: the new :py:class:`~katana.property_graph.PropertyGraph`
+        :returns: the new :py:class:`~katana.property_graph.Graph`
         """
         path_str = <string>bytes(str(path), "utf-8")
         with nogil:
             pg = handle_result_PropertyGraph(
```
Comment (gurbinder533) [inline python/katana/local/_graph.pyx:322]: pg-->g? and handle_result_PropertyGraph --> handle_result_graph?

File: python/katana/local/_graph.pyx [hunk 4]
```diff
@@ -352,7 +352,7 @@ cdef class PropertyGraph(PropertyGraphBase):
         cdef shared_ptr[_PropertyGraph] pg = make_shared[CGraph._PropertyGraph](
```
Comment (gurbinder533) [inline python/katana/local/_graph.pyx:352]: Can we not rename "_PropertyGraph"?

File: python/katana/local/analytics/_betweenness_centrality.pyx [hunk 5]
```diff
@@ -55,7 +55,7 @@ cdef extern from "katana/analytics/betweenness_centrality/betweenness_centrality
 
     Result[void] BetweennessCentrality(_PropertyGraph* pg, string output_property_name, const BetweennessCentralitySources& sources, _BetweennessCentralityPlan plan)
```
Comment (gurbinder533) [inline python/katana/local/analytics/_betweenness_centrality.pyx:56]: same.

File: python/katana/local/analytics/_independent_set.pyx [hunk 6]
```diff
@@ -25,7 +25,7 @@ from libcpp.string cimport string
 from katana.cpp.libgalois.graphs.Graph cimport _PropertyGraph
```
Comment (gurbinder533) [inline python/katana/local/analytics/_independent_set.pyx:25]: _PropertyGraph --> _Graph?

File: python/katana/local/analytics/_jaccard.pyx [hunk 7]
```diff
@@ -25,7 +25,7 @@ from libcpp.string cimport string
 from katana.cpp.libgalois.graphs.Graph cimport _PropertyGraph
```
Comment (gurbinder533) [inline python/katana/local/analytics/_jaccard.pyx:25]: same

File: python/katana/local/analytics/_k_truss.pyx [hunk 8]
```diff
@@ -27,7 +27,7 @@ from libcpp.string cimport string
 from katana.cpp.libgalois.graphs.Graph cimport _PropertyGraph
```
Comment (gurbinder533) [inline python/katana/local/analytics/_k_truss.pyx:27]: same

File: scripts/bench_python_cpp_algos.py [hunk 9]
```diff
@@ -23,14 +23,14 @@ def time_block(run_name):
     print(f"[TIMER] Time to run {run_name} : {round((timer_algo_end - timer_algo_start), 2)} seconds")
 
 
-def check_schema(property_graph: PropertyGraph, property_name):
+def check_schema(property_graph: Graph, property_name):
```
Comment (gurbinder533) [inline scripts/bench_python_cpp_algos.py]: property_graph --> graph?

**Review by denish44 — APPROVED** [review]

File: scripts/bench_python_cpp_algos.py (hunk 9 above)
Comment (gurbinder533) [reply scripts/bench_python_cpp_algos.py]: Throughout the script?

**Review by gurbinder533 — APPROVED** [review]
A few minor comments. Let's try to get this merged as soon as possible since conda package is updated I think and scripts are not, so it is causing some issues.

File: python/katana/__init__.py (hunk 1 above)
Comment (arthurp) [reply python/katana/__init__.py:38]: I just needed a module in which to stick all the global/static state accessors in the Katana libraries. I didn't know what else to call it. I'm open to suggestions. Would `katana_globals` be better?

File: python/katana/local/_graph.pyx (hunk 2 above)
Comment (arthurp) [reply python/katana/local/_graph.pyx:303]: No `_PropertyGraph` maps to the C++ class `PropertyGraph`, so I didn't want to rename it.

File: python/katana/local/_graph.pyx (hunk 3 above)
Comment (arthurp) [reply python/katana/local/_graph.pyx:322]: As above, those names refer to the C++ class.

File: python/katana/local/_graph.pyx (hunk 4 above)
Comment (arthurp) [reply python/katana/local/_graph.pyx:352]: As above. It's the C++ class.

File: python/katana/__init__.py (hunk 1 above)
Comment (gliga) [reply python/katana/__init__.py:38]: `katana_globals` is good. Also if always used with katana.SOMETHING, then you can remove katana twice:  `katana.globals` or `katana.static`.

File: python/katana/local/_graph.pxd [hunk 10]
```diff
@@ -12,7 +12,7 @@ cdef _convert_string_list(l)
 #
 # Python Property Graph
 #
-cdef class PropertyGraphBase:
+cdef class GraphBase:
```
Comment (gliga) [inline python/katana/local/_graph.pxd:15]: interesting to see the differences; like in Java would always be AbstractSOMTHING

File: python/katana/local/analytics/_wrappers.pyx [hunk 11]
```diff
@@ -37,17 +37,17 @@ cdef unique_ptr[NUMAArray[uint64_t]] handle_result_unique_numaarray_uint64(Resul
     return move(res.value())
 
 
-# "Algorithms" from PropertyGraph
+# "Algorithms" from Graph
 
-cdef extern from "katana/PropertyGraph.h" namespace "katana" nogil:
+cdef extern from "katana/Graph.h" namespace "katana" nogil:
```
Comment (gliga) [inline python/katana/local/analytics/_wrappers.pyx]: double checking if .h was renamed

**Review by gliga — APPROVED** [review]
Nice change.

I just realized I did not think much prior to this change about the lack of context when people review for code they do NOT see.  Namely if something changes, we look mostly add diff, but not at the sufficient context to check if something else might have to change.  Anyway, that is a separate discussion.

File: python/katana/local/analytics/_wrappers.pyx (hunk 11 above)
Comment (arthurp) [reply python/katana/local/analytics/_wrappers.pyx]: It was not. Ooops. How did this even build?!?! :-/ I'll fix this.

File: python/katana/local/_graph.pxd (hunk 10 above)
Comment (arthurp) [reply python/katana/local/_graph.pxd:15]: I was not really following a convention here. I should have I guess. I don't know what the Python convention is, or if there is a Python convention. Python does not have the well known and well defined conventions that Java does.

**Pull request merged**